### PR TITLE
add task in settings

### PR DESCRIPTION
**Pull request by charles-lebrun**
This pull request to solve the issue https://github.com/blueshiftone/streamdeck-toggl/issues/3 and allow to add task in settings

also fix some parameter who are legacy in the Toogle API

### CODE REVIEW

File: pi/main_pi.js [hunk 1]
```diff
@@ -81,26 +91,49 @@ function setWorkspace () {
...
+  } catch (e) {
+    document.getElementById('taskWrapper').classList.add('hidden')
+  }
+}
+
 async function updateProjects (apiToken, workspaceId) {
   try {
     await getProjects(apiToken, workspaceId).then(projectsData => {
       document.getElementById('pid').innerHTML = '<option value="0"></option>'
-      document.getElementById('workspaceError').classList.add('hiddenError')
```
Comment (brentonrace) [inline pi/main_pi.js:88]: @charles-lebrun Was it intentional to remove this line and similar on 103? My understanding is this is what hides the "Please select a workspace!" message that appears when you first add an API token.

Comment (brentonrace) [reply pi/main_pi.js:88]: Everything else looks legit to me, so once this is resolved I'll give it a quick test locally then can merge. Thanks for making this change!

Comment (NickCotter01) [reply pi/main_pi.js:88]: @charles-lebrun @brentonrace. Sorry to bump, but I use this plugin everyday and would be stoked to see this merge accepted. Have you had any movement on this issue?

Comment (brentonrace) [reply pi/main_pi.js:88]: @NickCotter01 I've been caught up in a project so was hoping Charles would push a fix for this comment, but I should have time at end of this week to tidy it up and merge.

Comment (brentonrace) [reply pi/main_pi.js:88]: @NickCotter01 Change is merged and I've posted a new release.

**Pull request merged**